.devcontainer/vmr-source-build/init.sh

```diff
@@ -19,11 +19,7 @@ git -C "$installer_dir" fetch --all --unshallow
 
 # We will try to figure out, which branch is the current (PR) branch based off of
 # We need this to figure out, which VMR branch to use
-vmr_branch=$(git -C "$installer_dir" log --pretty=format:'%D' HEAD^ \
-  | grep 'origin/'    \
-  | head -n1          \
-  | sed 's@origin/@@' \
-  | sed 's@,.*@@')
+vmr_branch=release/8.0.1xx
```

Comment:
This seems unrelated. Was the previous logic not working?

Reply:
The previous logic for some reason detected `main` so the synchronization took very long. I didn't notice this commit also made it here, will revert